runtime/src/main/java/org/corfudb/runtime/view/LayoutView.java

```diff
@@ -56,6 +62,76 @@ public int getQuorumNumber() {
         return (getLayout().getLayoutServers().size() / 2) + 1;
     }
 
+
+    public Layout quorumLayoutRead(List<String> layoutServers) throws QuorumUnreachableException {
```

Comment:
should the quorumLayoutRead verify from the acquired layout that all list of LayoutServers is complete for quorum computation?  Imagine the case where runtime is initialized with one layoutServer (for bootstrap purposes, but there are actually 3) and we call the quorumLayoutRead on the list of layoutServers given by the runtime, we might get an invalid quorum read success which is not real as all layout servers are not really queried (this is seen for example in the test verifyStateTransferAndMerge)...

We should also add some unit test to cover this case...